docs/4.1/admin-guide.md

```diff
@@ -1991,6 +1991,9 @@ belong to:
 $ tctl users add joe --k8s-groups="system:masters"
 ```
 
+!!! tip "NOTE":
+    In order to execute the above command, a Teleport role in RBAC needs to be created as `k8s-group`. An example RBAC role can be found in the [Kubernetes SSH](https://gravitational.com/teleport/docs/kubernetes_ssh/#impersonation) section.
```

Comment:
I wonder if we should just create this user by default and provide both `admin` and `k8s-group` as a role.  Otherwise how will OSS users create the role?

Reply:
Ow, I see in the docs for K8s-group to work we recommend using the Github Connector.